mentors/ruby/a.json

```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "Full time backend engineer. Rubyist."
-  }
+  }, 
+    {
+    "github_username": "arnoFleming",
+    "name": "Arno Flemingd",
```

Comment:
I'm pretty sure you know how to spell your name, but this looks suspicious...

Reply:
sheesh it does. These characters aren't even close... meh